compiler_opt/rl/trainer_test.py

```diff
@@ -92,7 +92,9 @@ def test_training(self):
         tf.compat.v1.train.AdamOptimizer(),
         num_outer_dims=2)
     test_trainer = trainer.Trainer(
-        root_dir=self.get_temp_dir(), agent=test_agent)
+        root_dir=self.get_temp_dir(),
+        agent=test_agent,
+        tensorboard_data_interval=1)
```

Comment:
It will fail on this revision because I forgot to change the variable name here (will fix). I forget the exact details, but with all the mocking that is setup for that test, I think it only runs ten iterations or something and then expects tensorboard data to be written after that, which it isn't getting because the new default summary export interval is significantly longer. I can rework the test if you want, but I felt that just changing the parameter was sufficient while still having the test touch on all the functionality that it needs to.

Reply:
yeah, just changing the parameter is good enough, the mock thing in the unit test is intended to make it a "unit test" :)